src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift

```diff
@@ -34,7 +34,7 @@ class AppInformationViewController: DynamicTableViewController {
 			.section(
 				header: .space(height: 32),
 				footer: .view(footerView()),
-				separators: false,
+				separators: .none,
```

Comment:
very nice to replace the bool with an adequate case here! 🏅

Reply:
😁